dist/imgix-js-core.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for imgix-core-js.js
+// Type definitions for @imgix/js-core.js
```

Comment:
```suggestion
// Type definitions for imgix-js-core.*.js
```

I would also accept just `@imgix/js-core` without the `.js`

Reply:
Nice catch!!!